src/fbw/src/FlyByWireInterface.cpp

```diff
@@ -327,8 +326,8 @@ void FlyByWireInterface::setupLocalVariables() {
   engineEngine1Timer = make_unique<LocalVariable>("A32NX_ENGINE_TIMER:1");
   engineEngine2Timer = make_unique<LocalVariable>("A32NX_ENGINE_TIMER:2");
 
-  idFlapsHandleIndex = make_unique<LocalVariable>("A32NX_FLAPS_HANDLE_INDEX");
-  idFlapsHandlePercent = make_unique<LocalVariable>("A32NX_FLAPS_HANDLE_PERCENT");
+  flapsHandleIndexFlapConf = make_unique<LocalVariable>("A32NX_FLAPS_CONF_INDEX");
+  flapsPosition = make_unique<LocalVariable>("A32NX_LEFT_FLAPS_ANGLE");
```

Comment:
For now yes, but all the external use of flaps will have to be updated at some point, probably when FWS will be up. I'm not sure what the plane is using as input in various systems, as it could be real angle / real configuration / desired configuration.
The left versus right case is even more complex because it would only be different in a failure case that we are far from being able to handle yet.  So for now Left or right is the same. In hydraulics I usually do (left+right) / 2

Reply:
Ok, that's fair enough then!